soc/xtensa/nxp_adsp/rt6xx/Kconfig.defconfig.series

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2023 Google LLC.
+# Copyright (c) 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_NXP_RT6XX
+
+config SOC_SERIES
+	string
+	default "rt6xx"
+
+config SOC_TOOLCHAIN_NAME
+	string
+	default "nxp_rt6xx"
```

Comment:
Is this toolchain available in zephyr-sdk?
I'm not aware of it.

Can we use the `nxp_rt500_adsp` the one used for RT585?

Have you compiled this support? How?
Probably with xcc, right?

Reply:
Indeed it is not.
I haven't explored compilation using the said toolchain for the RT595, as I was instructed to just focus on the port itself by my supervisor. My guess is that it could cause some problems, as the ISAs are for sure different (HiFi 4 vs. Fusion F1).

I used `xt-clang`, as `xcc` is not even present in newer versions of the Cadence toolchain.

Reply:
> Indeed it is not. I haven't explored compilation using the said toolchain for the RT595, as I was instructed to just focus on the port itself by my supervisor. My guess is that it could cause some problems, as the ISAs are for sure different (HiFi 4 vs. Fusion F1).
> 
Oh, you're right.

> I used `xt-clang`, as `xcc` is not even present in newer versions of the Cadence toolchain.

Most probably will need a toolchain for this target in zephyr sdk.
See examples: https://github.com/zephyrproject-rtos/sdk-ng/commit/7d1b95ecfa5b9ec45319eb4dda27dd6ce9bb94dc

Reply:
@VitekST  we will definitely need toolchain support in sdk-ng for your platform. As @iuliana-prodan  pointed the process can be easily done following an older platform addition. 

Mind that it will take some time until you merge the toolchain support and then we update the CI to use the new SDK. Expect 2-3 weeks on that if you follow up with maintainers once you submit your changes. 

We can help you with that, but you should just look at how we enabled rt595 or imx8ulp.

Without the toolchain support in the sdg-ng we cannot merge this PR as we need CI to compile it first.

Reply:
Just to be clear- since the imx8ulp has the same HiFi4 DSP, can we reuse the existing toolchain? Or will we need a new toolchain for this HiFi4 specifically?

Reply:
@danieldegrasse I haven't studied the differences between the two instances of the DSP, but from what I've encountered, they can differ, as the Xtensa IP is a customizable one. Luckily, @iuliana-prodan told me over internal Teams that adapting GCC with related tools and libraries should be just a matter of collecting a few source files from Cadence's BSP, creating a new crosstool-ng overlay with it and compiling the toolchain.

Reply:
@dbaluta @iuliana-prodan Toolchain PR created: https://github.com/zephyrproject-rtos/sdk-ng/pull/721